internal/security/setup/option/constant.go

```diff
@@ -27,14 +26,15 @@ const (
 
 	pkiSetupVaultJSON             = "pkisetup-vault.json"
 	pkiSetupKongJSON              = "pkisetup-kong.json"
+	pkiInitScratchDir             = "scratch"
+	pkiInitGeneratedDir           = "generated"
 	resourceDirName               = "res"
 	configTomlFile                = "configuration.toml"
 	envXdgRuntimeDir              = "XDG_RUNTIME_DIR"
-	defaultXdgRuntimeDir          = "/tmp"
+	defaultWorkDir                = "/tmp"
 	pkiInitBaseDir                = "/edgex/security-secrets-setup"
```

Comment:
since this is no longer the prefix for all the paths that use this variable, I'd prefer to remove the leading `/` from this variable so it's clear looking at the constant definition that this is not meant to be used as a prefix, instead it needs something else to be used as a prefix (i.e. `/run/edgex/security-secrets-setup` or `/tmp/edgex/security-secrets-setup`)